### PR TITLE
feat(ips): init ips react app

### PR DESCRIPTION
**Pull request by qpavy**
| Question         | Answer
| ---------------- | ---
| Branch?          | `project/ip-revamp`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #MANAGER-15657
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- ~~[ ] Only FR translations have been updated~~
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- ~~[ ] Breaking change is mentioned in relevant commits~~

### CODE REVIEW

**Review by darsene — COMMENTED** [review]
Can you fix e2e tests, they should work by default once app is generated

**Review by darsene — DISMISSED** [review]

**Review by anooparveti — CHANGES_REQUESTED** [review]
Since we need to remove e2e configs from generator, it's redundant. Can you remove it manually?

File: packages/manager/apps/ips/package.json [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    "@tanstack/react-query-devtools": "^5.51.21",
+    "axios": "^1.1.2",
+    "clsx": "^1.2.1",
+    "i18next": "^23.8.2",
+    "i18next-http-backend": "^2.4.2",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-i18next": "^14.0.5",
+    "react-router-dom": "^6.3.0",
+    "tailwindcss": "^3.4.4"
```
Comment (anooparveti) [inline packages/manager/apps/ips/package.json:45]: This can be moved to devDeps

File: packages/manager/apps/ips/playwright.config.ts
Comment (anooparveti) [inline packages/manager/apps/ips/playwright.config.ts:1]: Playwright is not required as well.

File: packages/manager/apps/ips/src/data/api/ips.ts [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  isAdditionalIp: any;
+  /** Filter the value of routedTo.serviceName property (like) */
+  routedToserviceName: any;
+  /** Filter the value of type property (&#x3D;) */
+  type: any;
+  /** Filter the value of version property (&#x3D;) (alpha) */
+  version: any;
+};
+
+export const getipListQueryKey = ['get/ip'];
```
Comment (anooparveti) [inline packages/manager/apps/ips/src/data/api/ips.ts:22]:
```suggestion
export const getIpListQueryKey = ['get/ip'];
```

File: packages/manager/apps/ips/src/data/api/ips.ts [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+/**
+ * List the ip.Ip objects : Your OVH IPs
+ */
+export const getipList = async (params: GetipListParams): Promise<any> =>
```
Comment (anooparveti) [inline packages/manager/apps/ips/src/data/api/ips.ts:27]:
```suggestion
export const getIpList = async (params: GetipListParams): Promise<any> =>
```

File: packages/manager/apps/ips/src/data/api/ips.ts [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+  apiClient.v6.get('/ip', { data: params });
+
+export type GetipIpParams = {
+  /**  */
+  ip?: any;
+};
+
+export const getipIpQueryKey = (params: GetipIpParams) => [
```
Comment (anooparveti) [inline packages/manager/apps/ips/src/data/api/ips.ts:35]:
```suggestion
export const getIpIpQueryKey = (params: GetipIpParams) => [
```

File: packages/manager/apps/ips/src/data/api/ips.ts [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+  `get/ip/${params.ip}`,
+];
+
+/**
+ * Your IP : Get this object properties
+ */
+export const getipIp = async (params: GetipIpParams): Promise<any> =>
```
Comment (anooparveti) [inline packages/manager/apps/ips/src/data/api/ips.ts:42]:
```suggestion
export const getIp = async (params: GetipIpParams): Promise<any> =>
```

**Review by mhelhali-soufien — DISMISSED** [review]

**Review by anooparveti — COMMENTED** [review]
If `mocks` folder is not used, can you remove it?

File: packages/manager/apps/ips/package.json [hunk 6]
```diff
@@ -0,0 +1,56 @@
+{
+  "name": "@ovh-ux/manager-ips-app",
+  "version": "0.0.0",
+  "private": true,
+  "description": "ips manager app",
```
Comment (anooparveti) [inline packages/manager/apps/ips/package.json:5]:
```suggestion
  "description": "Ips manager app",
```

File: packages/manager/apps/ips/package.json [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+    "dev": "tsc && vite",
+    "start": "lerna exec --stream --scope='@ovh-ux/manager-ips-app' --include-dependencies -- npm run build --if-present",
+    "start:dev": "lerna exec --stream --scope='@ovh-ux/manager-ips-app' --include-dependencies -- npm run dev --if-present",
+    "start:watch": "lerna exec --stream --parallel --scope='@ovh-ux/manager-ips-app' --include-dependencies -- npm run dev:watch --if-present"
+  },
+  "dependencies": {
+    "@ovh-ux/manager-config": "*",
+    "@ovh-ux/manager-core-api": "*",
+    "@ovh-ux/manager-core-utils": "*",
+    "@ovh-ux/manager-react-components": "*",
```
Comment (anooparveti) [inline packages/manager/apps/ips/package.json:24]: To use v1 of MRC
```suggestion
    "@ovh-ux/manager-react-components": "^1.14.1",
```

File: packages/manager/apps/ips/src/tracking.constant.ts [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+  CA: {
+    config: {
+      level2: '81',
+    },
+  },
+  US: {
+    config: {
+      level2: '81',
+    },
+  },
```
Comment (anooparveti) [inline packages/manager/apps/ips/src/tracking.constant.ts:16]: Refer -> https://github.com/ovh/manager/blob/596170e7b496f367ad75ab8273e6308d27e7b7a3/packages/manager/apps/dedicated/client/app/at-internet.constants.js
```suggestion
  EU: {
    config: {
      level2: '85',
    },
  },
  CA: {
    config: {
      level2: '85',
    },
  },
  US: {
    config: {
      level2: '57',
    },
  },
```

**Review by sachinrameshn — APPROVED** [review]

**Review by darsene — APPROVED** [review]

**Pull request merged**